sdk/Sdk.Generators/Extensions/ISymbolExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
+{
+    internal static class ISymbolExtensions
```

Comment:
nit/question: is this an extension class for "ISymbol"? /does it conform to naming convention to have a class start with an "I" (rather than only interfaces starting with "I")?

Reply:
I think it's fine? Perhaps if there was a class called "Symbol" there could be confusion (this is an extension for the ISymbol interface, not the Symbol class). 

I didn't name this class though - I just moved the file for clean up. It's originally from a PR from @kshyju [here](https://github.com/Azure/azure-functions-dotnet-worker/pull/830/files#diff-0516363d44bf8c20c2be9b928484562de1f64266398c5026d3db3dd6d6db7f43).